app/client/packages/design-system/widgets/src/components/DatePicker/src/DatePicker.tsx

```
@@ -0,0 +1,70 @@
+    description,
+    errorMessage,
+    isLoading,
+    isRequired,
+    label,
+    size = "medium",
+    ...rest
+  } = props;
+
+  return (
+    <HeadlessDatePicker
+      aria-label={Boolean(label) ? undefined : "DatePicker"}
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)